tests/samples/matmul_fill_spec_pad.mlir

```diff
@@ -1,135 +0,0 @@
-// This script shows an example lowering matmul through IREE for a special accelerator.
```

Comment:
lets not delete it? we may want to keep it for those who want to experiment with it.